src/qibolab/platforms/abstract.py

```diff
@@ -105,9 +106,8 @@ def __init__(self, name, runcard):
         self.relaxation_time = None
         self.sampling_rate = None
 
-        self.native_single_qubit_gates = {}
-        self.native_two_qubit_gates = {}
-        self.two_qubit_natives = set()
+        self.single_qubit_natives = {}
+        self.two_qubit_natives = TwoQubitNatives(0)
```

Comment:
I am not very familiar with `Flag` objects but why do you need to initialize with 0?

Reply:
I start with empty flag object and then i add the two qubit native gates reading from the runcard (line 142)

Reply:
Yes, I was just curious why the initialization is `TwoQubitNatives(0)` and not just `TwoQubitNatives()` but anyway that's not very important.